tests/unit/layout-test.js

```diff
@@ -60,23 +59,23 @@ describe('layout', function () {
       a: {x: 75 / 2, y: 100 / 2},
       b: {x: 75 / 2, y: 100 + 150 + 70 + 150 + 200 / 2}
     })
-    expect(_.pick(g.edge('a', 'b'), ['x', 'y']))
+    expect(['x', 'y'].reduce((acc, key) => ({...acc, [key]: g.edge('a', 'b')[key]}), {}))
       .eqls({x: 75 / 2, y: 100 + 150 + 70 / 2})
   })
 
   describe('can layout an edge with a long label, with rankdir =', function () {
-    _.forEach(['TB', 'BT', 'LR', 'RL'], function (rankdir) {
+    ['TB', 'BT', 'LR', 'RL'].forEach(rankdir => {
       it(rankdir, function () {
         g.graph().nodesep = g.graph().edgesep = 10
-        g.graph().rankdir = rankdir
-        _.forEach(['a', 'b', 'c', 'd'], function (v) {
+        g.graph().rankdir = rankdir;
```

Comment:
semicolon in front of next line?

Reply:
yes

Reply:
I had not encountered that particular problem before.